umbraco/Umbraco.CMS.Backoffice
# Feature: Collection Toolbar Filter Field

## Description

Since reviewing the Dictionary Collection toolbar filter (PR #2415) last week, I looked at the other Collection toolbars to see if we could de-duplicate the filter input code, to make a reusable component.

This PR introduces a `<umb-collection-filter-field>` component.

## Types of changes


## How to test?

Try out the Collection toolbar filter field on the following areas, to make sure that it still works as expected.

- Dictionary
- Documents
- Extension Insights
- Media
- Members
- Users
- User Groups
